app/promotion_keys.py

```diff
@@ -25,10 +25,11 @@ def post(self):
         if showtime is None:
             return self.error(404, "Show time not found.")
 
-        promotion_key = yield create_promotion_key(showtime_id)
-        self.set_header("Location", "{0}/api/promotion_keys/{1}".
-                        format(CONFIG.get("base_url"), promotion_key))
-        self.api_response({'promotion_key': promotion_key}, 201)
+        promotion_keys = []
+        for i in range(0, count):
+            promotion_keys.append((yield create_promotion_key(showtime_id)))
```

Comment:
Not something you need to fix at all, but just so you know, you can do this:

```
promotion_keys = yield [create_promotion_key(showtime_id) for _ in range(count)]
```

You also don't need to provide the lower bound on a `range` unless it's _not_ 0.